examples/lib/openstreetmap.js

```diff
@@ -0,0 +1,47 @@
+var vumigo = require('vumigo_v02');
+var location = require('../../lib');
+var LocationState = location.LocationState;
+var OpenStreetMap = location.providers.openstreetmap.OpenStreetMap;
+
+var App = vumigo.App;
+var EndState = vumigo.states.EndState;
+
+var LocationApp = App.extend(function(self){
+    App.call(self, 'states:start');
+
+    self.states.add('states:start', function(name){
+        // This state will get the location data from the user and store it in
+        // the contact store. We will store and return the longitude and
+        // latitude information to the user.
```

Comment:
@hodgestar Sorry to comment after the merge, things went a bit quick on this.  This comment says that it stores the longitude and latitude, but I don't think that's actually implemented.  Perhaps the intention was that `self.extract_address_data` would by default also return the longitude and latitude?  Shall I open another issue to implement this?